benchmarks/benchmarks/stats.py

```diff
@@ -8,11 +8,17 @@
     pass
 
 from .common import Benchmark
-try:    
+try:
     from scipy.stats._distr_params import distcont
 except ImportError:
     pass
 
+try:
+    from itertools import compress
+except ImportError:
+    pass
+
+
```

Comment:
```suggestion
from itertools import compress
```
`itertools` is standard library. No chance of error.